lib/tasks/braintree_tokens.rake

```diff
@@ -0,0 +1,8 @@
+namespace :champaign do
+  desc "Create braintree payment method tokens out of existing token columns"
+  task braintree_tokens: :environment do
+    Payment::BraintreeCustomer.all.each do |customer|
+      Payment::BraintreePaymentMethod.create!({customer_id: customer.customer_id, token: customer.card_vault_token})
```

Comment:
There's no migration to remove the card_vault_token at the moment so this should work. We can clean up that column later.